tests/meta-data-from-ast-test.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const expect = require('chai').expect;
+const metaDataFromAST = require('../lib/meta-data-from-ast');
+const defaultParser = require('../lib/default-parser');
+
+function toAST(source) {
+  return defaultParser(source);
+}
+
+describe('.metaDataFromAST', function() {
+  describe('ES5', function() {
+    const FOO = toAST(`define('foo', ['a', 'b/c'], function() { });`);
+    const A = toAST(`define('a', ['exports', 'require'], function() { })`);
+    const C = toAST(`define('c', ['../a', '../d'], function() { })`);
+    const D = toAST(`define('d', ['foo'], function() { })`);
```

Comment:
these should either be inside the `it()` block or in a `beforeEach()` block

Reply:
If they are updated to be deep frozen is that also ok?